### DISCUSSION TITLE
AttributeError: 'Context' object has no attribute 'send_modal'

### DISCUSSION THREAD

**Discussion by ghost** [Pycord-Development/pycord#2397] [Q&A]
[closed]
why   (I'm cant verify phone number to join in discord server)

**Comment by Lulalaby** [comment]
Please join our server and ask the question in the appropriate channel. Make sure to read our rules and help rules first.
Just asking why, isn't helping anyone.

**Reply by ghost** [reply]
I can't verify my phone number

**Reply by Lulalaby** [reply]
What's your id

**Reply by ghost** [reply]
filass.ils
